core/src/components/popover/test/dismissOnSelect/popover.e2e.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+import { openPopover } from '../test.utils';
+
+test.describe('popover: dismissOnSelect', async () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/src/components/popover/test/dismissOnSelect');
+  });
+
+  test('should not dismiss a popover when clicking a hover trigger', async ({ page }) => {
+    const ionPopoverDidPresent = await page.spyOnEvent('ionPopoverDidPresent');
+    
+    await openPopover(page, 'hover-trigger');
+    const popover = page.locator('.hover-trigger-popover');
+    const hoverTrigger = page.locator('#more-hover-trigger');
+
+    await hoverTrigger.hover();
+    await ionPopoverDidPresent.next(); // wait for hover popover to open
+    await hoverTrigger.click();
+
+    // ensure parent popover is still open
+    expect(popover).toBeVisible();
+
+    await page.setIonViewport();
+    expect(await page.screenshot()).toMatchSnapshot(`
+      popover-hover-trigger-${page.getSnapshotSettings()}.png
+    `);
+  });
+
+  test('should not dismiss a popover when clicking a click trigger', async ({ page }, testInfo) => {
+    // TODO
+    test.skip(
+      testInfo.project.name === 'firefox',
+      'Parent popover disappears when click trigger is clicked. Cannot replicate locally. Needs further investigation.'
+    );
```

Comment:
[Screenshot from trace.](https://user-images.githubusercontent.com/90629384/165155247-f02b6669-c764-4cf8-9ce5-f0abfb403c21.png) Interestingly, the nested popover remains. I elected to leave this alone since we're skipping existing bugs for now, although I haven't been able to replicate locally. If this is acceptable, I'll make a tech debt ticket in Jira and add the number here.